docs/OPTIONS-LIST.md

```diff
@@ -379,6 +369,37 @@ see [notes E-](/docs/NOTES.md#e-) for details
 
 ##         B2) GENERIC GENERAL/RANKED/UNRANKED ARGUMENTS :
```

Comment:
You might want to add some explanation here why this is a separate section.
`The following settings can be set for all games, or for ranked/unranked seperatly.` Something along those lines.

Reply:
indeed, added here :+1: 
https://github.com/online-go/gtp2ogs/pull/234/commits/0baa6e7688ddb40b168deafd29de0de3b30086b0#diff-404fedfb31d78a21d007198a5d1985a1R195

Reply:
I think actually added here. :) https://github.com/online-go/gtp2ogs/pull/234/commits/d61f3fe02d6c7bef82f546b885b17872f784aa88